de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/ReferencePointNegotiation.java

```diff
@@ -75,7 +75,7 @@
 
     protected IReferencePointManager referencePointManager;
 
-    public ProjectNegotiation(final String id, final JID peer,
+    public ReferencePointNegotiation(final String id, final JID peer,
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Constructor has 10 parameters, which is greater than 7 authorized. ([squid:S00107](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS00107))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=259&issues=AWZfKMfGDZywvPe4LUfd&open=AWZfKMfGDZywvPe4LUfd)

[ISSUE_KEY]: AWZfKMfGDZywvPe4LUfd